src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -41,7 +41,7 @@ public Car create(Car car) {
 
     @Override
     public Optional<Car> get(Long id) {
-        String getCarByIdQuery = "SELECT car_id, model, name, manufacturer_id, country "
+        String getCarByIdQuery = "SELECT id, model, name, manufacturer_id, country "
```

Comment:
```suggestion
        String getCarByIdQuery = "SELECT id as car_id, model, name, manufacturer_id, country "
```
підозрюю що інакше цей запит не зможе коректно розпарситись, адже в методі getCarFromResultSet() ми дістаємо ід машини за назвою колонки car_id